src/app/components/scrollBar/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+const ScrollBarContainer = styled.div({
+  overflowY: 'hidden',
+  scrollbarGutter: 'stable',
+  ':hover': {
+    overflowY: 'auto',
+  },
+  '&::-webkit-scrollbar': {
+    width: 8,
+    height: 10,
+    background: 'transparent',
```

Comment:
why are we making the scrollbar transparent and taking up width, rather than display:none ?